modin/core/io/text/text_file_dispatcher.py

```diff
@@ -635,18 +634,19 @@ def check_parameters_support(
         -------
         bool
             Whether passed parameters are supported or not.
+        Optional[str]
+            `None` if parameters are supported, otherwise an error
+            message describing why parameters are not supported.
         """
         skiprows = read_kwargs.get("skiprows")
         if isinstance(filepath_or_buffer, str):
-            if not cls.file_exists(
-                filepath_or_buffer, read_kwargs.get("storage_options")
-            ):
-                return False
+            if not cls.file_exists(filepath_or_buffer):
```

Comment:
This if statement has been changed.
```suggestion
            if not cls.file_exists(filepath_or_buffer, read_kwargs.get("storage_options"):
```